wiki/scripts/build.sh

```diff
@@ -1,8 +1,42 @@
-# HOST=https://wiki.dgraph.io
-#HOST=http://localhost:8080
-HOST=http://34.208.35.222
-VERSION=0.7.3
-
-HUGO_TITLE="Dgraph Doc ${VERSION}" hugo\
-     --destination=public/latest\
-     --baseURL=$HOST/latest
+#!/bin/bash
+
+HOST=https://docs.dgraph.io
+
+rebuild() {
+	echo "$(date) Updating docs for branch: $1"
+	# Generate new docs after merging.
+	HUGO_TITLE="Dgraph Doc ${2}" hugo\
+		--destination=public/$2\
+		--baseURL=$HOST/$2 1> /dev/null
+}
+
+checkAndUpdate()
+{
+	local branch=$1
+	git checkout -q $1
+	local tag=$2
+
+	UPSTREAM=$(git rev-parse "@{u}")
+	LOCAL=$(git rev-parse "@")
+
+	if [ $LOCAL != $UPSTREAM ] ; then
+		git merge -q origin/$branch
+		rebuild $branch $tag
+	fi
+
+	if [ ! -d "public/$tag" ] ; then
+		rebuild $branch $tag
+	fi
+}
+
+# Lets move to the Wiki directory.
+cd /home/ubuntu/dgraph/wiki
+
+while true; do
+	echo -e "Starting to check branches\n"
+	git remote update > /dev/null
```

Comment:
Should we also pull latest theme e.g. `(cd themes/hugo-doc && git remote update)`?

Related idea:
should themes be managed as a submodule in case there is a breaking change in theme?